requirements/dev.txt

```diff
@@ -1,17 +1,16 @@
-autopep8 ~= 1.4
-camel-case-switcher ~= 2.0
-coverage ~= 5.4
-flake8 ~= 5.0.4
-hypothesis ~= 6.2
-mypy ~= 0.812
-m2r2 ~= 0.2
-nose ~= 1.3
-PyYAML ~= 5.4
-pytest ~= 6.2
-pytest-cov ~= 2.11
-pytest-mock ~= 3.5
-pytest-watch ~= 4.2
-responses ~= 0.12
+autopep8
+camel-case-switcher
```

Comment:
What's the thinking behind removing the versions?

Reply:
don't bother reviewing this yet - I have made several commits trying to resolve snyk issues without breaking everything with no luck. Unfortunately I don't know how to run snyk on the python deps without pushing to github.